week3_model_free/sarsa.ipynb

```diff
@@ -395,7 +395,7 @@
     "\n",
     "Here are some of the things you can do if you feel like it:\n",
     "\n",
-    "* Play with epsilon. See learned how policies change if you set epsilon to higher/lower values (e.g. 0.75).\n",
+    "* Play with an epsilon. See how learned policies change if you set an epsilon to the higher/lower values (e.g. 0.75).\n",
```

Comment:
It's a very specific epsilon, so it should be "the".

On the other hand, you may regard "epsilon" as a name, in which case there should be no article.